### ISSUE TITLE
Thunderbird 122 beta - cannot add new tabs to toolbar

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#439]
When we drag a new folder to the QuickFolders toolbar, this does not add a new tab. Instead we see the following message in error console:

```
Uncaught TypeError: MailServices.accounts.FindAccountForServer is not a function
    addFolder chrome://quickfolders/content/quickfolders-model.js:81
    addFolder chrome://quickfolders/content/quickfolders.js:1028
    drop chrome://quickfolders/content/quickfolders.js:1052
    ondrop chrome://messenger/content/messenger.xhtml:1
quickfolders-model.js:81:38

```

**Comment by RealRaven2000** [comment]
In newer versions of Thunderbird this (and other) uppercase function(s) have been converted to camel case. I have modified the code accordingly to be forward compatible.

```js
// pattern to stay compatible with both ESR 115 and 122 beta:
let account = (MailServices.accounts.FindAccountForServer || MailServices.accounts.findAccountForServer)(folder.server).key;

```
[QuickFolders-mx-6.4pre93.zip](https://github.com/RealRaven2000/QuickFolders/files/13518609/QuickFolders-mx-6.4pre93.zip)

----

...to install the test version, download the zip file and then drag the file into Thunderbird Add-ons Manager (do not extract contents).

**Comment by RealRaven2000** [comment]
Fixed in 6.4 released on 04/12/2023
